bareos/files/bareos-config.jinja

```diff
@@ -16,23 +16,35 @@
         {%- elif v is string  %}
 {{ n|indent(ind, True) }}{{ operator }}{{ v }}
         {%- else %}
-            {%- for value in v %}
-{{ n|indent(ind, True) }}{{ operator }}{{ value }}
-            {%- endfor %}
+{{ _list(n, v, ind) }}
         {%- endif %}
     {%- endfor %}
 {%- endmacro -%}
+{%- macro _list(x, y, ind1, operator=' = ') -%}
+        {%- for var in y %}
+                {%- if var is mapping %}
+{{ x|indent(ind1, True) }} {
+{{ _config(var, ind1+ind_increment) }}
+{{ '}'|indent(ind1, True) }}
+                {%- elif var is iterable and var is not string %}
+{{ _list(x,var, ind1) }}            
+                {%- else %}
+{{ x|indent(ind1, True) }}{{ operator }}{{ var }}
+        {%- endif %} 
```

Comment:
A silly detail, but mind you indent in this `{% endif %}` to match the rest of the block?